### PR TITLE
Adapt root consistency threshold

### PR DESCRIPTION
**Pull request by NicoleJurjew**
I adapted the threshold for the TOF-root consistency test to 3.3 (was: 2.5) as this is the lowest number with one significant figure for which the test passes.

### CODE REVIEW

File: examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py [hunk 1]
```diff
@@ -110,6 +110,67 @@ def PointCloud3D(DataHandler):
...
+
+def PointCloud3D_all(point_sources_data):
+    # import matplotlib.pyplot as plt
+    # import random
+
+    fig = plt.figure(figsize=(10, 10))
+    ax = fig.add_subplot(projection='3d')
+    
+    for key in point_sources_data.keys():
+        DataHandler = point_sources_data[key]
```
Comment (robbietuk) [inline examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py]: objects/variables should be `snake_case`, `PascalCase` is generally reserved for classes.

File: examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py [hunk 2]
```diff
@@ -110,6 +110,67 @@ def PointCloud3D(DataHandler):
     ax.set_ylabel('y (mm)')
     ax.set_zlabel('z (mm)')
     ax.legend(['Origin and Tolerance', 'Mean coords and stddev ', 'Voxel Positions'])
+    
+    plt.show()
+
+def PointCloud3D_all(point_sources_data):
+    # import matplotlib.pyplot as plt
+    # import random
```
Comment (robbietuk) [inline examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py]: I know I haven't thus far with this script (I should), do you mind adding some documentation? [example](https://www.programiz.com/python-programming/docstrings)

File: src/recon_test/test_consistency_with_root.cxx [hunk 3]
```diff
@@ -207,7 +207,7 @@ setup()
 
   // Failure conditioner and recording
   nonTOF_distance_threshold = 1.5 * norm(grid_spacing); // Using norm(grid_spacing) as a nonTOF_distance_threshold
-  TOF_distance_threshold = 2.5 * norm(grid_spacing); // Using norm(grid_spacing) as a tof_distance_threshold
+  TOF_distance_threshold = 3.3 * norm(grid_spacing); // Using norm(grid_spacing) as a tof_distance_threshold
```
Comment (robbietuk) [inline src/recon_test/test_consistency_with_root.cxx]: Can you update the comment here? Why `3.3`?

**Review by robbietuk — CHANGES_REQUESTED** [review]
Awesome job, a few small comments.

File: examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py (hunk 1 above)
Comment (NicoleJurjew) [reply examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py]: Hi Robbie, thanks for your comments! I changed the names in the original function (def PointCloud3D(DataHandler)) accordingly, I hope it's more consistent with naming conventions now.

File: examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py (hunk 2 above)
Comment (NicoleJurjew) [reply examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py]: Let me know if this is not sufficient, I'm never sure what to write there.

File: src/recon_test/test_consistency_with_root.cxx (hunk 3 above)
Comment (NicoleJurjew) [reply src/recon_test/test_consistency_with_root.cxx]: After having changed source positions in GATE and seeing the test-results didn't change dramatically, we agreed that I should use the lowest number for which the test passes. That's what I put in the comments now. I hope this is sufficient, please let me know if I should add more info. Thanks!

**Comment by robbietuk** [comment]
Awesome, thank you 😄

**Pull request merged**